src/components/ItemDetails.js

```diff
@@ -0,0 +1,59 @@
+/* eslint-disable dot-notation */
+/* eslint-disable quotes */
+import React from 'react';
+import { useParams } from 'react-router';
+import { useSelector } from 'react-redux';
+
+const ItemDetails = () => {
+  const countriesList = useSelector((state) => state.covidReducer.oneCountry);
+  const { name } = useParams();
+  const countries = countriesList ? countriesList.find((country) => country.name === name) : "Still fetching";
+  const {
+    confirmedCases,
+    recovered,
+    openCases,
+    allDeaths,
+    id,
+  } = countries;
+  return (
+    <div className="detail-info" key={id}>
+      <h1>{name}</h1>
+      <table className="table">
+        <thead>
+          <tr className="row">
+            <th className="column head">Confirmed</th>
+            <th className="column head">Deaths</th>
+            <th className="column head">Open</th>
+            <th className="column head">Recovered</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr className="row">
+            <td className="column">
+              {confirmedCases}
+              {' '}
+              cases
+            </td>
+            <td className="column">
+              {allDeaths}
+              {' '}
+              cases
+            </td>
+            <td className="column">
+              {openCases}
+              {' '}
+              cases
+            </td>
+            <td className="column">
+              {recovered}
+              {' '}
+              cases
+            </td>
+          </tr>
+        </tbody>
+      </table>
+    </div>
```

Comment:
Let's stick to the required design template 👍 

Your details page design


>![Screenshot (346)](https://user-images.githubusercontent.com/66112760/147087093-50a7c776-81b3-4f89-b97b-b7d0da07c4a8.png)


Design template
>![Screenshot (342)](https://user-images.githubusercontent.com/66112760/147086623-82733235-7898-44d1-90e2-57ab460b462f.png)

Reply:
done